### ISSUE TITLE
Request Object Passed by value Flow is not working

### ISSUE THREAD

**Issue by Maliesha** [wso2/product-is#13295]
**Describe the issue:**
Request object passed by value flow is not giving the expected outcome.

**How to reproduce:**
Follow the steps in blog [1]

**Result**

When we decode the retrieved id_token, can't see the customClaim1 value which we marked as essential:true under id_token in the request object payload.

<img width="572" alt="Screenshot 2022-03-22 at 12 01 11 PM" src="https://user-images.githubusercontent.com/47919820/159422270-15423e14-5273-4cc8-99d2-fe23d4c357de.png">

**Expected behavior:**

When we decode the retrieved id_token, we should be able to see that customClaim1 value which we marked as essential:true under id_token in the request object payload has been retrieved with the id_token.

<img width="570" alt="Screenshot 2022-03-22 at 12 01 43 PM" src="https://user-images.githubusercontent.com/47919820/159422243-5259d457-24ec-4a8f-817a-a754bee06431.png">

**Environment information:**
 - Product Version: 5.12.0.alpha 16 (reproduced in alpha 15 as well)
 - OS: Mac

[1] [https://shanikaw.medium.com/working-with-request-object-in-wso2-identity-server-42156dcfcab7](https://shanikaw.medium.com/working-with-request-object-in-wso2-identity-server-42156dcfcab7)

**Comment by Maliesha** [comment]
With the change [wso2-extensions/identity-inbound-auth-oauth#1623](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1623) done for [#3443](https://github.com/wso2-enterprise/asgardeo-product/issues/3443), we are not returning any claims for openid scope. 
We didn't get any claims in the request object path too since we tried with openid scope. 
I could able to get the claims in the id token when I tried with some custom oidc scopes. Hence this is not an issue in the product.
